docs/modules/operation/pages/deep-dive/kafka-producer/configure-kafka.adoc

```diff
@@ -169,3 +173,25 @@ admin@opennms()> config:update
 ----
 
 In the example above, we set the vertex and edge topics to be different by default.
+
+
+== Configure the alarm sync streams client
+
+When producing alarms to a topic, we also automatically enable a Kafka Stream client to help synchronize the contents of the topic and ensure eventual consistency with the database.
+
+The streams client takes different properties than the producer and requires a separate configuration map.
```

Comment:
"streams client" or "stream client"? it's singular when mentioned in the previous sentence. (and captialized)